test-automation/api-tests/package.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
harmny-sdk is nodejs 10.0 min, so please enforce that here engineStrict key, also please open pr to harmony-js sdk itself to enforce the nodejs 10 min requirement as well: